code/game/objects/effects/spiders.dm

```diff
@@ -6,20 +6,62 @@
 	anchored = 1
 	density = 0
 	health = 5
+	var/burning
+	var/burn_count = 0
+	var/burn_overlay = "web_burning"
 
 //similar to weeds, but only barfed out by nurses manually
 /obj/effect/spider/ex_act(severity)
 	switch(severity)
 		if(1.0)
 			qdel(src)
 		if(2.0)
-			if (prob(50))
+			if (prob(75))
 				qdel(src)
 		if(3.0)
-			if (prob(5))
+			if (prob(50))
 				qdel(src)
 	return
 
+/obj/effect/spider/Destroy()
+	STOP_PROCESSING(SSobj, src)
```

Comment:
You sure you want to do this?
Im p-sure they dont process by default so by having this here when it gets destoryed and not processing it can lead to some issues

Reply:
`STOP_PROCESSING` already checks for `is_processing`, but i can add a redundant check to be safe